src/test/ui/wf/hir-wf-check-erase-regions.rs

```diff
@@ -0,0 +1,14 @@
+// Regression test for #87549.
+// compile-flags: -C incremental=true
```

Comment:
I think this flag takes a path to the incremental for, not a boolean.

Reply:
You are right, of course.